kas/include/nxp.yml

```diff
@@ -0,0 +1,29 @@
+header:
+  version: 14
+
+repos:
+  meta-freescale:
+    url: https://github.com/Freescale/meta-freescale.git
+    branch: scarthgap
+  meta-freescale-3rdparty:
+    url: https://github.com/Freescale/meta-freescale-3rdparty.git
+    branch: scarthgap
+  meta-freescale-distro:
+    url: https://github.com/Freescale/meta-freescale-distro.git
+    branch: scarthgap
+
+  meta-mender-community:
+    layers:
+      meta-mender-nxp:
+
+distro: poky
+
+local_conf_header:
+  nxp: |
+    ACCEPT_FSL_EULA = "1"
+    MENDER_BOOT_PART_SIZE_MB = "64"
+    MENDER_STORAGE_TOTAL_SIZE_MB = "4096"
+    IMAGE_FSTYPES:remove = " rpi-sdimg"
+    MENDER_FEATURES_ENABLE:append = " mender-uboot mender-image-sd"
+    MENDER_FEATURES_DISABLE:append = " mender-grub mender-image-uefi"
+    EXTRA_IMAGE_FEATURES = "debug-tweaks"
```

Comment:
Please remove.

Reply:
OK, I remove `debug-tweaks` from `nxp.yml` and add it to `olimex-imx8mp-evb.yml`. For Olimex i.MX8MP SoM and EVB the targeted image is `core-image`base`. Without `debug-tweaks`, `core-image-base` defaults to “root” without any password which prevents from manually logging and performing a Mender update.

Reply:
Well all images default to `root` and don't allow login. I add the `debug-tweaks` feature through a personal kas config add-on in my builds, just like `DL_DIR`, `SSTATE_DIR` and my Mender tenant token. Please remove it completely from the PR, as I want to avoid it leaking into actual product build configurations - people are notorious for just copying and shipping stuff. We should show proper security hygiene at least in the default configurations :-)

Reply:
OK, I'll remove it. Keep in mind that as of the moment `include/raspberrypi.yml` also adds `debug-tweaks`: https://github.com/mendersoftware/meta-mender-community/blob/scarthgap/kas/include/raspberrypi.yml#L24

P.S. Please, don't look the git blame for this line in `raspberrypi.yml` :) I am innocent :)

Reply:
Very good catch, thanks for pointing it out. A clear oversight at my end :-(